packages/rpc-core/src/rpc-methods/index.ts

```diff
@@ -114,8 +122,23 @@ export type SolanaRpcMethods = GetAccountInfoApi &
     RequestAirdropApi &
     SendTransactionApi &
     SimulateTransactionApi;
+type SolanaRpcMethodsWithAirdrop = SolanaRpcMethodsWithoutAirdrop & RequestAirdropApi;
 
-export function createSolanaRpcApi(config?: Config): IRpcApi<SolanaRpcMethods> {
+export type SolanaRpcMethodsDevnet = IRpcApiMethodsDevnet<SolanaRpcMethodsWithoutAirdrop>;
+export type SolanaRpcMethodsTestnet = IRpcApiMethodsTestnet<SolanaRpcMethodsWithAirdrop>;
+export type SolanaRpcMethodsMainnet = IRpcApiMethodsMainnet<SolanaRpcMethodsWithAirdrop>;
```

Comment:
Might be able to completely avoid this with https://github.com/solana-labs/solana-web3.js/pull/1974#discussion_r1431991541